ors-api/src/main/java/org/heigit/ors/api/ORSEnvironmentPostProcessor.java

```diff
@@ -45,8 +45,9 @@ public void postProcessEnvironment(ConfigurableEnvironment environment, SpringAp
             log.info("Configuration file set by environment variable.");
         }
         if (configLocations.isEmpty()) {
+            String home = System.getenv("HOME");
```

Comment:
This is OS-dependent. On Windows, you might be looking for `HOMEPATH` or `HOMEDRIVE` or something.
Using [System.getProperty()](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/lang/System.html#getProperties()) might be the way to go here.
```suggestion
            String home = System.getProperty("user.home");
```

Otherwise, we could (if `HOME` is not set) determine that we're not in an environment that might use `~/.config/…`, and by extension probably not use `/etc/…` and only add the first configLocation and report accordingly.